src/ossl/aes.rs

```diff
@@ -2632,9 +2636,20 @@ impl AesCmacOperation {
         buf.zeroize();
 
         #[cfg(feature = "fips")]
-        {
-            self.fips_approved = check_mac_fips_indicators(&mut self.ctx)?;
-        }
+        self.fips_approval_cmac()?;
+
+        Ok(())
+    }
+
+    #[cfg(feature = "fips")]
+    fn fips_approval_cmac(&mut self) -> Result<()> {
+        /* The MAC len needs to have specific length for FIPS
+         * NIST SP 800-38B A.2:
+         * > For most applications, a value for Tlen that is at least 64
+         * should provide sufficient protection against guessing attacks.
+         */
+        self.fips_approved = Some(self.maclen >= (32 / 8));
```

Comment:
Is it intended to use 32 or 64 bits here? The comment seems to reference 64 bits but the code uses 32 bits.

Reply:
Indeed. Thanks for double-checking.